server/pkg/webapp/serve.go

```diff
@@ -148,7 +151,67 @@ func serveConfig(r chi.Router, config webappConfig) {
 	r.Get(options.CleanBase(config.appUrl, "custom.css"), func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could we please also consider doing two additional bits?

1. make SASS compiler optional. For a lot of cases (mainly development and _light usage_), it's simply not needed and takes needles effort to set up
2. make SCSS asset path configurable to enable users uploading their own source files. Something similar we've done with auth templates -- it could come handy to users and I don't imagine this taking a lot of effort to enable.